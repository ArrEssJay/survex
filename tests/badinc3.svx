; pos=fail
; include of a file which includes a file which includes a nonexistent file
*include badinc2
