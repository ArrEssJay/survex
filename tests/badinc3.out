In file included from ./badinc3.svx:3:
In file included from ./badinc2.svx:3:
./badinc.svx:3: Couldn't open file "nonexistent_file"
cavern: No survey data
